ocs_samples/basic_samples/Authentication/ClientCredentialFlow/DotNet/ClientCredentialFlow/README.md

```diff
@@ -12,7 +12,7 @@ Replace the placeholders in the [apsettings](./apsettings.json) file with your T
 ### Using Visual Studio
 
 - Load the .csproj
-- Rebuild solution
+- Rebuild project
 - Run it
   - If you want to see the token and other outputs from the program, put a breakpoint at the end of the main method and run in debug mode
 
```

Comment:
I know this was a missing before you got to this readme, but can you add whatever blurb you say for running the test in this readme as well?